### PR TITLE
Add ISCE3 Installation Script, Help Script Subcommand Dispatcher, and OnDemand System Activity Tracker

### PR DESCRIPTION
**Pull request by nerivera**
This PR introduces several significant features and improvements to the project, as part of the work completed during my internship at JPL. The changes include the addition of an installation script for ISCE3, a subcommand dispatcher for a help script, and an activity tracker for the OnDemand System running on the CI machine (Jenkins).

### CODE REVIEW

File: activity_tracker/1-download-data.sh
Comment (jplzhan) [inline activity_tracker/1-download-data.sh:1]: Please adjust the bucket name here to be an environment variable.

File: activity_tracker/2-update-data.py [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+to provide the NISAR Ondemand Science service. \n\
+\n\
+Reminder on how to shutdown your instance: \n\
+    1. Log into the NISAR Ondemand Science system via-> https://nisar.jpl.nasa.gov/ondemand \n\
+    2. Click File -> Hub Control Panel \n\
+    3. Click \"Stop my server\" \n\
+\n\
+********************************************************************************************** \n\
+* This is an automated email notification system. Do not reply back to this email as this inbox is not monitored. \n\
+* If you have questions, please email the nisar-ondemand-help@jpl.nasa.gov address for assistance.                 \n\
```
Comment (jplzhan) [inline activity_tracker/2-update-data.py]: Please adjust the email here to be an environment variable.

File: activity_tracker/3-upload-data.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+aws s3 cp ./usage-history.csv s3://nisar-st-data-ondemand/Ondemand_Logs/Activity_Tracker/usage-history.csv
```
Comment (jplzhan) [inline activity_tracker/3-upload-data.sh]: Please adjust the bucket names here to be an environment variable.

File: activity_tracker/2-update-data.py [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+    - Started: {started} [{started_hours}hr(s) ago] \n\
+\n\
+It is suggested that someone from the ondemand-help team reach out \
+to the user directly to investigate account idling or log into their \
+account to exam if there are background processes running to determine if it \
+should be shut it down by an admin. \
+\n\
+********************************************************************************************** \n\
+* This is an automated email notification system. Do not reply back to this email as this inbox is not monitored. \n\
+* If you have questions, please email the nisar-ondemand-help@jpl.nasa.gov address for assistance.                 \n\
```
Comment (jplzhan) [inline activity_tracker/2-update-data.py]: Please change this email name to be an environment variable.

**Review by jplzhan — CHANGES_REQUESTED** [review]
There are a couple places we would like to remove the hardcoding of bucket names and the help email, which will make it easier to adapt this script to other OnDemand systems if these names change.

**Comment by nerivera** [comment]
I've made the requested changes. Please test them on the CI machine and let me know if there's any issues (I no longer have access to it).

**Review by jplzhan — APPROVED** [review]

**Pull request merged**